stack/app/ecr.tf

```diff
@@ -0,0 +1,8 @@
+module "ecr" {
```

Comment:
the ECR module is not environment aware, i.e. if we add 2 stacks for the same project, one stack per environment (e.g. "staging" and "production"), then we will have some collision here?

so far, we've opted to have 1 container registry per application, and use that for every environment.

WDYT @samkahchiin ?